src/TLuaInterpreter.cpp

```diff
@@ -1546,12 +1464,7 @@ int TLuaInterpreter::addMapMenu(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#removeMapMenu
 int TLuaInterpreter::removeMapMenu(lua_State* L)
 {
-    if (!lua_isstring(L, 1)) {
-        lua_pushfstring(L, "removeMapMenu: bad argument #1 type (Menu name as string expected, got %s!)", luaL_typename(L, 1));
-        return lua_error(L);
-    }
-    QString uniqueName = lua_tostring(L, 1);
-
+    QString uniqueName = getVerifiedString(L, __func__, 1, "Menu name");
     if (uniqueName == "") {
```

Comment:
We do not want unneeded raw string literals - even if they are empty.
```suggestion
    if (uniqueName.isEmpty()) {
```

Reply:
That line wasn't even touched at all or part of original scope, but I'll give you this one, as it's the only `if (x == "")` in the whole file...